packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/cart-api/set-attributed-staff-to-line-item.tsx

```diff
@@ -0,0 +1,25 @@
+
+import React from 'react';
+import {
+  reactExtension,
+  useApi,
+  Tile
+} from '@shopify/ui-extensions-react/point-of-sale';
+
+const SmartGridTile = () => {
+  const api = useApi<'pos.home.tile.render'>();
+
+  return (
+    <Tile 
+      title="My App" 
```

Comment:
```suggestion
      title='My App' 
```

Just realized all of these are double-quotes, and the rest are singles.